doc/sphinx/conf.py.in

```diff
@@ -169,6 +169,9 @@ man_pages = [
     ("docs/fpga_tools/pci_device/pci_device", 'pci_device', u'Common operations for PCIe devices', [author], 8),
     ("docs/fpga_tools/opae.io/opae.io", 'opae.io', u'User space access to PCIe devices', [author], 8),
     ("docs/fpga_tools/host_exerciser/host_exerciser.md", 'host_exerciser', u'Host Exerciser lpbk and memory', [author], 8),
+    ("docs/fpga_tools/hssi_ethernet/hssistats.md", 'hssistats', u'High Speed Serial Interface ethernet statistics', [author], 8),
+    ("docs/fpga_tools/hssi_ethernet/hssimac.md", 'hssimac', u'High Speed Serial Interface ethernet max', [author], 8),
+    ("docs/fpga_tools/hssi_ethernet/hssilpbk.md", 'hssiloopback', u'High Speed Serial Interface ethernet loopback', [author], 8),
```

Comment:
s,hssiloopback,hssilpbk, to match the markdown name.

Reply:
fixed,  name as file name and script hssiloopback